assets/performanceprofile/scripts/set-rps-mask.sh

```diff
@@ -1,18 +1,16 @@
 #!/usr/bin/env bash
 
-full_path=${1}
-[ -n "${full_path}" ] || { echo "The full device path argument is missing" >&2 ; exit 1; }
+path=${1}
+[ -n "${path}" ] || { echo "The device path argument is missing" >&2 ; exit 1; }
 
 mask=${2}
 [ -n "${mask}" ] || { echo "The mask argument is missing" >&2 ; exit 1; }
 
-# replace '-' with '/'
-de_escape_path="/sys${full_path//-//}"
+queue_path=${path%rx*}
 
-# get the path for the queues
-queues_path=${de_escape_path%/rx*}
+queue_num=${path#*queues/}
+# replace '/' with '-'
+queue_num="${queue_num/\//-}"
```

Comment:
minor nit: is this substitution required? If I read correctly $path is not escaped anymore and this is basically de-escaping the queue nr part